index.js

```diff
@@ -3637,6 +3637,10 @@ export const createContainerSpecLikeBom = async (path, options) => {
     path,
     (options.multiProject ? "**/" : "") + "*.yml"
   );
+  let dfFiles = getAllFiles(
+    path,
+    (options.multiProject ? "**/" : "") + "*Dockerfile"
```

Comment:
Add another * at the end. For example, see cdxgen naming conventions under ci.

Reply:
Containerfile is used by podman and other tools.

Reply:
Great feeedback.  Updated to support `Containerfile` and wider pattern expression.